tests/e2e/verification_post_day1_test.go

```diff
@@ -148,18 +148,22 @@ var _ = Describe("Verify cluster", func() {
 			Expect(len(getResp.Body().AWS().Tags())).To(Equal(len(buildInTags)))
 		}
 	})
-	It("can be imported - [id:65684]",
+	It("can be imported - [id:65684][id:72961]",
```

Comment:
The same as above one. Maybe we can clean up the cases in polarion?

Reply:
updated